### PR TITLE
Introduce webpack dev server

### PR DESCRIPTION
**Pull request by mlunoe**
Here it is running in a cloud9 instance:

I did not get to get the hot reloading to work, so you'll have to refresh you page during development

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,141 +1,140 @@
...
-                            <span class="match-details__team__player">Michael</span>
-                            <span class="match-details__team__player">Christian</span>
-                            <span class="match-details__team__points">12 points</span>
-                        </div>
-                    </a>
-                </article>
-            </section>
-        </main>
-    </body>
+  <head>
```
Comment (mlunoe) [inline src/index.html]: Only change here is indentation

File: src/js/__tests__/example.test.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    };
+
+    this.result = this.testFunction();
+  })
+
+  it('should have the result computed', function () {
+    expect(this.result).equal(3);
+  });
+
+  it('should call testFunction', function () {
```
Comment (mlunoe) [inline src/js/__tests__/example.test.js]: Moved this file to be under `__tests__` and added another test with spy

File: src/styles/main.scss [hunk 3]
```diff
@@ -157,7 +157,7 @@ html, body {
       TYPOGRAPHY
 *********************/
 
-@import url(http://fonts.googleapis.com/css?family=Open+Sans:400italic,400,300,600,700);
+@import url(//fonts.googleapis.com/css?family=Open+Sans:400italic,400,300,600,700);
```
Comment (mlunoe) [inline src/styles/main.scss]: Use the protocol that we are running the application on, so when you run `https` you don't get errors

**Comment by rcorral** [comment]
🚢  after comments are taken careof

**Comment by mlunoe** [comment]
A few comments after pair programming session:
- Debugging breakpoints and source maps was not working properly (properly related)
- Add babel-loader to have es6 templates. Will something be too easy? I don't think so.

File: webpack.config.js [hunk 4]
```diff
@@ -0,0 +1,119 @@
+var autoprefixer = require('autoprefixer');
+var HtmlWebpackPlugin = require('html-webpack-plugin');
+var webpack = require('webpack');
+var path = require('path');
```
Comment (mlunoe) [inline webpack.config.js]: Done

File: webpack.config.js [hunk 5]
```diff
@@ -0,0 +1,119 @@
+var autoprefixer = require('autoprefixer');
+var HtmlWebpackPlugin = require('html-webpack-plugin');
```
Comment (mlunoe) [inline webpack.config.js]: Removed

**Comment by mlunoe** [comment]
@rcorral should we merge this guy?

**Pull request merged**